prepare_mkl.sh

```diff
@@ -75,16 +75,16 @@ MXNET_ROOT=`dirname $0`
 USE_MKLML=0
 # NOTE: if you update the following line, please also update the dockerfile at
 # tests/ci_build/Dockerfile.mkl
-VERSION_MATCH=20170908
+VERSION_MATCH=20171227
 PLATFORM=$(uname)
 if [ $PLATFORM == "Darwin" ]; then
     INFIX=mac
 elif [ $PLATFORM == "Linux" ]; then
     INFIX=lnx
 fi
-ARCHIVE_BASENAME=mklml_${INFIX}_2018.0.20170908.tgz
+ARCHIVE_BASENAME=mklml_${INFIX}_2018.0.1.20171227.tgz
```

Comment:
It would help to use ${VERSION_MATCH} here so that the version needs to be defined in only one place.

All other changes look fine to me.

Reply:
Sure, I will update the PR as you suggested.